ncbi_genome_download/__main__.py

```diff
@@ -2,9 +2,9 @@
 import argparse
 import logging
 
-from . import __version__
-from . import download
-from . import EDefaults as dflt
+from ncbi_genome_download import __version__
+from ncbi_genome_download import download
+from ncbi_genome_download import EDefaults as dflt
```

Comment:
Where are you seeing this as a problem? That bit wasn't the issue for me.
Rather `download(vars(args))` clearly needs to be `download(**vars(args))`, and then you also need to remove the "debug", "verbose", and "retries" entry.